addons/l10n_eg/data/account_tax_report_data.xml

```diff
@@ -194,7 +194,7 @@
         </field>
     </record>
     <record id="tax_report_withholding_tax" model="account.report">
-        <field name="name">2. Withholding Tax</field>
+        <field name="name">WH Tax</field>
```

Comment:
Are we sure we don't want `Withholding Tax` instead ?

Reply:
I don't have a strong opinion, so if he says/prefer that way, I'm ok with it (I just find it more ugly and don't see the point into changing that)